td-logger/src/logger.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+
+//! A standalone logger to directly log messages, independent of the log crate.
+
+use core::fmt::{self, Write};
+use lazy_static::lazy_static;
+use spin::Mutex;
+
+use crate::{dbg_port_write, dbg_write_string};
+
+pub const LOG_LEVEL_VERBOSE: usize = 4;
```

Comment:
Nit: since we have a lot of log level bits to play with, should this be split into `LOG_LEVEL_DEBUG` and `LOG_LEVEL_TRACE`? That would align with the log crates levels:

- https://docs.rs/log/0.4.14/log/enum.Level.html

Reply:
fixed